packages/website/docs/_samples/main/List/GroupHeaders/sample.html

```diff
@@ -13,16 +13,16 @@
 
     <ui5-list selection-mode="Multiple">
         <ui5-li-groupheader>Front End Developers</ui5-li-groupheader>
-        <ui5-li image="../assets/images/avatars/woman_avatar_3.png" icon="navigation-right-arrow"
+        <ui5-li icon="navigation-right-arrow"
```

Comment:
At least in the new samples (website/docs/_samples), let's keep the images, it's good to show that after the "image" property is removed, we can get the same thing as before via the "image" slot.